coco docs: clarify platforms page and agent policy (#479)
NVIDIA/cloud-native-docs: confidential-containers/attestation.rst

* Using sealed secrets
* Requesting secrets directly from workloads

Attestation is necessary but not sufficient for a secure end-to-end deployment.
The Kata shim on the host remains untrusted and can still issue Agent API calls into the guest

> unless you restrict them with a Kata agent security policy.

Author: nit: pronouns are generally bad and it's better to restate the noun.  Unfortunately, this is a bit of a guess:

```suggestion
unless you restrict the Agent API calls that the host can make with a Kata agent security policy.
```

Reviewer: Ack - also addressing same pattern in `confidential-containers/configure-workloads.rst`

* Using sealed secrets
* Requesting secrets directly from workloads

> A complete attestation process should also cover the Kata Agent API surface: which operations the

Author: ```suggestion
A complete attestation process manages the Kata Agent API surface: which operations the
```

This gist here is that "should" is the kiss of death in tech docs and is best replaced with more decisive language.  Please sub in the correct verb in place of "manages", maybe "constrains" or "limits."

Reviewer: Agree'd, thanks for catching this.
